aten/src/ATen/native/transformers/cuda/mem_eff_attention/gemm/custom_mma_base.h

```diff
@@ -88,7 +88,7 @@ class CustomMmaBase {
       Shape::kN / WarpGemm::kN,
       Shape::kK / WarpGemm::kK>;
 
-  /// Number of warp-level GEMM oeprations
+  /// Number of warp-level GEMM operations
```

Comment:
Hmm, technically it's a 3rd party code (though placed at a very wrong location), so it should stay as close to the source as possible, shouldn't it?

Reply:
done